uber/models/mivs.py

```diff
@@ -86,8 +86,8 @@ class IndieStudio(MagModel):
     completed_discussion = Column(Boolean, default=False)
     read_handbook = Column(Boolean, default=False)
     training_password = Column(UnicodeText)
-    selling_at_event = Column(Boolean, nullable=True)
-    needs_hotel_space = Column(Boolean, nullable=True)
+    selling_at_event = Column(Boolean, nullable=True, admin_only=True)  # "Admin only" preserves null default
```

Comment:
I had to blow away my development environment and start from scratch, which meant creating new MIVS groups for testing. When I did this, I found to my dismay that `selling_at_event` and `needs_hotel_space` were defaulting to False when a MIVS group was created -- this didn't affect us this year because the columns were added after the fact, but it would have hit us late next year with no good way to fix it. 😬

The reason they were getting set to False is because they were 'unrestricted' Boolean columns. Our `apply` function assumes that any unrestricted Boolean column actually exists on the attendee's form, and so if it's not in the parameters, it should be set to False because the checkbox must be unchecked. That's not true in this case.

We handle these columns explicitly in the page handlers that do pertain to them, so we don't need them to be unrestricted for when the MIVS studios are filling out their checklist.